src/gtc/unstructured/usid_codegen.py

```diff
@@ -318,7 +316,7 @@ class UsidGpuCodeGenerator(UsidCodeGenerator):
                 return;
             % if len(prim_sid.entries) > 0:
             auto ${ prim_sid.ptr_name } = ${ prim_sid.origin_name }();
-            gridtools::sid::shift(${ prim_sid.ptr_name }, gridtools::device::at_key<
+            gridtools::sid::shift(${ prim_sid.ptr_name }, gridtools::host_device::at_key<
```

Comment:
this can stay 'device' I think

Reply:
Yes, you are right. On the other hand, this is more consistent with the code that is generated in the next lines (i.e. from the common part of the code generator).